examples/amp-access-fewcents.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
```

Comment:
Should this example use the new amp-access-fewcents component?

Reply:
these access components are a bit weird in that they don't actually have their own `<amp-access-xxx>` custom elements. You might have been confused by validator file, which I think is incorrect.

Reply:
Ah, OK, thanks.